photutils/psf.py

```diff
@@ -808,11 +813,21 @@ def psf_photometry(data, positions, psf, fitshape=None,
                 fit_infos.append(fitter.fit_info)
             if fit_messages is not None:
                 fit_messages.append(fitter.fit_info['message'])
+            if param_uncert:
+                uncert.append(np.sqrt(np.diag(fitter.fit_info['param_cov'])))
```

Comment:
I'm not sure all fitter will have a `'param_cov'` and even if currently all fitters have that, I don't think that that is going to be true in the future e.g. some MCMC fitters might return full distribution functions instead of a single number. Obviously, if they don't python is going to throw an error, but for user friendlyness, I suggest to catch that error here and re-raise is with a useful message; i.e. instead or "KeyError('param_cov')" the user would get "Fitter does not return uncertainties in 'param_cov" keyword." or something similar.